Gemfile

```diff
@@ -23,11 +23,12 @@ group :checks do
 end
 
 # Excluded from CI except on latest MRI Ruby, to reduce compatibility burden
-group :docs do
+group :docs, optional: true do
   gem "dokaz", "~> 0.0.5"
   gem "redcarpet", "~> 3.6"
   gem "yard", "~> 0.9"
   gem "yard-doctest", "~> 0.1"
+  gem "webrick"
```

Comment:
I notice all the other gems here have versions specified. I don't _think_ it matters, but just double-checking there's no reason (e.g. something dependabot-related) to have it here?

Reply:
Added webrick minor version like the others